projects/power_distribution/README.md

```diff
@@ -0,0 +1,33 @@
+# Power Distribution
+
+## What is the purpose of the board?
+A power distribution board is necessary to supply power to the various subsystems in the vehicle. This board performs distribution, startup, shutdown and emergency shutdown as required in regulations. In addition to supporting powering the boards, it also powers components directly, such as the horn, camera and others.

```

Comment:
The horn is actually powered by UV cutoff (which is controlled by rear PD) - probably good to mention UV cutoff somewhere.